src/Commands/RendersTenants.php

```diff
@@ -83,8 +86,11 @@ protected function renderTenantCredentials(\Slides\Saml2\Models\Tenant $tenant)
     {
         $this->output->section('Credentials for the tenant');
 
+        $spEntityIdOverrideStatus = (!empty($tenant->sp_entity_id_override) ? ' (Manual override)' : '');
```

Comment:
Took me a few seconds to understand this ternary. I think it read better inverting the condition and dropping the brackets.

```suggestion
        $spEntityIdOverrideStatus = empty($tenant->sp_entity_id_override) ? '' : ' (Manual override)';
```